crates/matrix-sdk-ui/src/timeline/date_dividers.rs

```diff
@@ -599,7 +604,14 @@ impl Display for DateDividerInvariantsReport<'_, '_> {
             }
 
             writeln!(f, "\nFinal state:")?;
-            write_items(f, self.final_state.iter().cloned().collect::<Vec<_>>().as_slice())?;
+            write_items(
+                f,
+                self.final_state
+                    .iter_remotes_and_locals_regions()
```

Comment:
Again, maybe we want to write out the full state including the start event. Or maybe not - not sure.